ha/const.py

```diff
@@ -89,3 +90,8 @@
 PCS_CLUSTER_STATUS="pcs cluster status"
 PCS_CLUSTER_UNSTANDBY="pcs cluster unstandby --all"
 PCS_STATUS_NODES="pcs status nodes"
+
+# Cluster manager
+CM_CONTROLLER_INDEX="controller_interface"
+CM_CONTROLLER_SCHEMA="{}/controller_interface.json".format(CONFIG_DIR)
+CM_ELEMENT=["cluster", "node", "service"]
```

Comment:
storageset should also be there?
It's a logical entity though.

Reply:
Done